Bluepill-cli/Bluepill-cli/Bluepill/Reporters/BPTreeParser.h

```diff
@@ -25,6 +25,8 @@
 // /export/home/tester/hudson/data/workspace/MP_TRUNKDEV_DISTRIBUTED_TEST/voyager-ios_7dee32c1fdb9facfff35737351eeab72cfa90126/Testing/VoyagerIntTestsLib/Shared/VoyagerIntTestCase.swift:172: error: -[VoyagerFeedIndividualPageTests.FeedEmptyFeedVariant1SplashTest testHighlightedDeepLinkEmptyFeedHidden] : The step timed out after 10.00 seconds: Waiting for notification "concurrent_dispatch_queue_finish"
 #define TEST_CASE_FAILED @"(.*):(\\d+): error: -\\[(.*) (.*)\\] : (.*)"
 
+#define UITEST_CASE_FAILED @".*Assertion Failure:([^:]*):(\\d+): (.*) from input.*"
```

Comment:
Is this the only regex we need to capture UI test failures ?

Reply:
I don't know. This fixes the problem in #200 as demonstrated by @PavloBasiuk on #206. If there are others we should add them to the test cases.

Reply:
Let me give it a try :)

Reply:
It's better now. System output from test-run is saved and available. But exception message is still lost:
Here is what I see in TEST-FinalReport.xml `<failure type="Failure" message="UNKNOWN ERROR - PARSING FAILED: XCTestOutputBarrierTest Case '-[ShareFileUITests.CreateFileTests testCreateSpreadsheet]' started.">
Unknown File:0
        </failure>`

And screenshot from Jenkins:
![screen shot 2017-12-07 at 10 44 28 am](https://user-images.githubusercontent.com/6704791/33723845-bbf0b296-db3b-11e7-8b00-5243c482f092.png)

Reply:
@PavloBasiuk thanks!

Reply:
`#define UITEST_CASE_FAILED @".*Assertion Failure:([^:]*):(\\d+): (.*)"` worked as a charm for me! I'd push changes directly to your branch but don't have access right 👍  

Thanks for fixing!!!

Reply:
updated!